apps/nextjs-website/src/app/profile/chatbot-history/page.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import { useTranslations } from 'next-intl';
+import { Stack, Typography } from '@mui/material';
+import React from 'react';
+
+const ChatbotHistory = () => {
+  const t = useTranslations('profile.chatbot');
```

Comment:
I know in some place we had used `useTranslations` this way but as convention we have decide to avoid adding params and use the complete path in `t(...)`. This make easier to check translation and to mantainn it.
```suggestion
  const t = useTranslations();
```